include/config/config_safeguards.h

```diff
@@ -88,7 +88,7 @@
 #endif
 
 // Enable the map select feature if booting to map select or using map select for file 1
-#if (BOOT_TO_MAP_SELECT || MAP_SELECT_ON_FILE_1) && !ENABLE_MAP_SELECT
+#if (BOOT_TO_DEBUG_OPENING || MAP_SELECT_ON_FILE_1) && !ENABLE_MAP_SELECT
```

Comment:
with this change if you set ``BOOT_TO_MAP_SELECT`` to true it won't enable the map select automatically anymore